docs/generated/comments.json

```
@@ -2960,7 +2952,7 @@
         "subheads": [],
         "description": "List all sources available in the running OBS instance",
         "return": [
-          "{Array of Objects} `sources` Array of sources as objects",
+          "{Array<Object>} `sources` Array of sources as objects",
```

```suggestion
          "{Array<Object>} `sources` Array of sources",
```
Can also be applied to other instances of this.